cross-modal-search/app.py

```diff
@@ -23,8 +23,7 @@ def config():
 
 
 def index_restful():
-    flow = Flow().load_config('flows/flow-index.yml')
-    flow.use_rest_gateway()
+    flow = Flow(protocol='http').load_config('flows/flow-index.yml', override_with={'protocol': 'http'})
```

Comment:
To keep the code simple

```suggestion
    flow = Flow().load_config('flows/flow-index.yml', override_with={'protocol': 'http'})
```
I'm now sure whether my revision can work, please test it.

Reply:
Yep, I believe the proposed change works for me. Please see [e11b980](https://github.com/jina-ai/examples/pull/672/commits/e11b980488645a32eac1fbe3a7c9ca27def8d850) for the change.